### ISSUE TITLE
opam init "cp" command not found

### ISSUE THREAD

**Issue by RuedigerLudwig** [ocaml/opam#4001]
Hello, 

I have Debian 18.04 and installed ocaml and opam via apt.

I have no alias for cp, and consider my basic installation to be quite clean. 

When I run opam init I get this output and no .opam is created:

```
$ opam init
Checking for available remotes: rsync and local, git, mercurial, darcs. Perfect!

=-=- Fetching repository information =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[default] synchronized from https://opam.ocaml.org
[NOTE] The repository 'default' will be *permanently* redirected to https://opam.ocaml.org/1.2.2 (opam-version < "2.0~")
[default] synchronized from https://opam.ocaml.org/1.2.2
[ERROR] Initialisation failed
"cp": command not found.
Fatal error:
"cp": command not found.
```

When I try to run opam init -V debug I get baiscally the same output at last four lines with no further hint what could have gone wrong.

Does anybody know what could be wrong here?

Thank you
Rüdiger

**Comment by dra27** [comment]
Did you delete `cp`, or it’s not installed in that basic an installation (that surprises me?). opam expects basic Posix utilities

**Comment by RuedigerLudwig** [comment]
> Did you delete `cp`, or it’s not installed in that basic an installation (that surprises me?). opam expects basic Posix utilities

I do not think so.

```
$ which cp
/bin/cp
```
and 
```
$ $SHELL --version
GNU bash, version 4.4.20(1)-release (x86_64-pc-linux-gnu)
```
I get the same results using opam init with or without sudo and there are no aliases for cp

**Comment by AltGr** [comment]
That's really puzzling... Maybe `PATH` got wiped at some point ? You could try running with `--debug -vvv`, but I am not sure it will help in this case.

**Comment by dra27** [comment]
Ah, sorry - I took from "I have no alias for cp" that you'd removed it for some reason! The package manager has installed opam 1.2.2, so you may have a little more joy upgrading it to opam 2 (see https://opam.ocaml.org/doc/Install.html#Ubuntu). Even if it has the same error, the debugging information may be more helpful.

**Comment by dbuenzli** [comment]
Also provide the output of `env`.

**Comment by rjbou** [comment]
For external command call, better run opam with verbose > 1, with `-vv` for ex (increasing `v`/verbosity reports more information). As said @dra27, upgrading to opam 2  would be more easy to find the problem origin. Note that opam uses `command -v <cmd>` to get the full path.

**Comment by RuedigerLudwig** [comment]
Okay, there seem to be furth issues with my installation here. I the same errors after I installed opam 2. When I tried to get the error logs out of my virtual machine I realized, that there seem to be similar errors there, but unrelated to opam. So I set up a new machine to try out everything there and all worked fine. 

I could dig deeper into the problems at the old machine, but whatever I might find, I am pretty sure it is unrelated to anything in opam. 

So unless you want me to get the error messages I would suggest to close this issue and put it onto something weird that happend on my computer.

Thank you for the help and suggestions. And sorry that I came up with something that seemingly did not have its origin here.
